src/Test/jTorrent/UrlToolsTest.java

```diff
@@ -0,0 +1,59 @@
+		String goodHash = "pokemon/ditto/";
+		boolean https = true;
```

Comment:
This enables https, which requires that we give the server we call some identity back with our requests. for now it is encoded directly into the request as "fakeIdentity". We might want to rethink this as part of #4